### ISSUE TITLE
Items can be moved to the end of container even if options.accepts returns false

### ISSUE THREAD

**Issue by bikutaa** [luckylooke/dragular#136]
Seems like an item can be moved to the end of a container even if `options.accepts` returns **false**.

e.g. Items should be not able to be moved anywhere with:

```
$scope.dragularOptions = {
  accepts: function() {
    return false;
  }
};
```

But it can be moved to the last position.

Example link: http://jsfiddle.net/bikutaa/21k4hyzL

**Comment by luckylooke** [comment]
Thanks for reporting and reproducing the issue, I will fix it soon.

**Comment by bikutaa** [comment]
Awesome, thanks for the quick reply. 
Looking forward to your fix, need it for a big project.

**Comment by luckylooke** [comment]
Fixed in latest version 4.4.1, here is fiddle with fix http://jsfiddle.net/21k4hyzL/4/
Thanks for cooperation ;)
